src/lib/layout-constants.js

```diff
@@ -0,0 +1,3 @@
+export default {
+    fullSizeMinWidth: 1096
```

Comment:
This one says 1096 and the other one says 1095. Is that right?

Reply:
Also noticed that things stick out a bit before min width UI kicks in, so we might want to do it sooner?
![image](https://user-images.githubusercontent.com/2855464/32572067-772ac6fc-c497-11e7-8352-5f29562058d1.png)